opensplice_cmake_module/cmake/Modules/FindOpenSplice.cmake

```diff
@@ -76,7 +79,7 @@ if(NOT _ospl_home STREQUAL "")
     "ddsserialization"
     "ddsuser"
     "ddsutil"
-    "durability"
+    # "durability"
```

Comment:
does that library not exist at all ? or just not built properly in our custom deb ?
I think we should either put a comment here or remove the line depending on what makes most sense